kolibri/plugins/management/assets/src/views/top-nav/index.vue

```diff
@@ -40,7 +40,7 @@
     $trs: {
       classes: 'Classes',
       data: 'Data',
-      configuration: 'Configuration',
+      settings: 'Settings',
```

Comment:
We should also update the page title, this page's h1, and the route.

![localhost_8000_management_facility_](https://user-images.githubusercontent.com/7193975/33030418-202ba960-cdd0-11e7-9386-9faa00de5e41.png)